appveyor.yml

```diff
@@ -0,0 +1,14 @@
+version: 1.0.{build}
+os: Windows Server 2012 R2
+environment:
+  PATH: C:\MinGW\bin;C:\msys64\usr\bin;%PATH%
+install:
+- cmd: >-
```

Comment:
I don't know actually, I made all the settings on the web GUI for appveyor and then downloaded the yml file based on the settings.